addons/account/static/src/components/section_and_note_fields_backend/section_and_note_backend.scss

```
@@ -2,40 +2,65 @@
 // The goal of this file is to contain CSS hacks related to allowing
 // section and note on sale order and invoice.
 
-table.o_section_and_note_list_view tr.o_data_row {
-    &.o_is_line_note {
-        font-style: italic;
-    }
-    &.o_is_line_section {
-        font-weight: bold;
-        background-color: #DDDDDD;
-        --table-bg: var(--300);
-    }
-    &.o_is_line_subsection {
-        --table-bg: var(--200);
-    }
-    &.o_is_line_section, &.o_is_line_subsection {
-        border-top: 1px solid #BBB;
-        border-bottom: 1px solid #BBB;
-        .o_list_section_options {
-            width: 1px;  // to prevent the column to expand
-            button {
-                padding: 0px;
-                background: none;
-                border-style: none;
-                display: table-cell;
-                cursor: pointer;
+table.o_section_and_note_list_view {
+    --o-section-border-color: #{$gray-500};
```

`section` is way to generic IMO. The component name is `SectionAndNoteListRenderer`, by following the guidelines it should be something like:

```suggestion
    --o-SectionAndNote-border-color: #{$gray-500};
```

edit: also in this case (see my next comment), to create a CSS variable feels unnecessary. You can create a SCSS one instead.